ingest/Dockerfile

```diff
@@ -4,4 +4,14 @@ LABEL com.connexta.application.name=cdr-ingest
 ARG JAR_FILE
 COPY ${JAR_FILE} /cdr-ingest
 ENTRYPOINT ["/cdr-ingest"]
-EXPOSE 8080
+EXPOSE 8080 10040 10050
+# Enable JMX so the can JVM be monitored
```

Comment:
❓Aren't these changes in another PR: https://github.com/connexta/multi-int-store/pull/111? I recommend keeping the changes in separate PRs. Does https://github.com/connexta/multi-int-store/pull/111 need to be merged first?